components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/SampleTestObjectCreator.java

```diff
@@ -333,7 +335,8 @@ public static API getMockApiSummaryObject() {
                 createdTime(LocalDateTime.now()).
                 createdBy(API_CREATOR).
                 apiDefinition(apiDefinition).
-                lastUpdatedTime(LocalDateTime.now());
+                lastUpdatedTime(LocalDateTime.now()).
+                securityScheme(3);
```

Comment:
Check if its correct to hard-code the security scheme in this manner.

Reply:
This method is used to get a mock API object. Setting security scheme to 3 allows both oauth and api key schemes. This was discussed offline before the PR was merged.